src/tooltipServiceWrapper.ts

```diff
@@ -97,53 +62,57 @@ class TooltipServiceWrapper implements ITooltipServiceWrapper {
         // Mouse events
         selection.on("mouseover.tooltip", () => {
             // Ignore mouseover while handling touch events
-            if (!this.canDisplayTooltip(event)) {
+            if (!this.canDisplayTooltip(d3Event))
                 return;
-            }
```

Comment:
This is not recommend, please always use curly brackets

Reply:
fixed